drivers/hwmon/Kconfig

```diff
@@ -1286,6 +1286,16 @@ config SENSORS_PWM_FAN
 	  This driver can also be built as a module.  If so, the module
 	  will be called pwm-fan.
 
+config SENSORS_RPI_POE_FAN
+	tristate "Raspberry Pi POE HAT fan"
+	depends on RASPBERRYPI_FIRMWARE
+	depends on THERMAL || THERMAL=n
+	help
+	  If you say yes here you get support for Raspberry Pi POE HAT fan.
+
```

Comment:
It would be nice to write down the meaning of the acronym at least in this help (Power over Ethernet).